ActindoWorkFlow/Overview/04_WorkflowProcessElements.md

```diff
@@ -36,54 +37,57 @@ Workflow processes, as well as the workflows itself, consists of certain element
 
 ![Transition](../../Assets/Screenshots/ActindoWorkFlow/Workflows/Action.png "[Transition]")
 
-The transitions act as building blocks that are chained together through places to configure a workflow. They are represented by a rectangle. Transitions are also called workflow actions.
+The transitions act as building blocks that are chained together through places to configure a workflow. They are used to process the incoming data and to output the processed data to the next place.  
+Transitions are represented by a rectangle. When a transition is executed in a workflow process, it is called action or process action. This means that an action is the actual execution of a configured transition.  
+For detailed information, see [Actions](#actions). 
 
-The transitions available to create workflows depend on the modules installed in the system. A certain number of transitions is always preconfigured in the *Workflows* module, the so-called *Core actions*, see [Workflow core actions](../UserInterface/08_CoreActions.md). Every API endpoint of the modules automatically becomes available as a transition and can be used in any workflow. However, you can also build your own transitions independent from the API. Some transition may have configuration options, but most of them do not.
+The transitions available to create workflows depend on the modules installed in the system. A certain number of transitions is always preconfigured in the *Process Orchestration* module, the so-called *Core actions*, see [Workflow core actions](../UserInterface/08_CoreActions.md). Each API endpoint of the installed modules is automatically available as a transition. It can be used in any workflow. However, you can also build your own transitions independent of an API. Some transition may have configuration options, but most of them do not.
```

Comment:
Ich würde "It can be used in any workflow" weglassen. Ich denke, es ist hier selbstverständlich.
"... build your own transitions independently of an API" (sollte es nicht ein Adverb sein?)